frontend/src/index.pug

```diff
@@ -175,14 +178,16 @@ html
             .summary-charts__title(v-if="filterGroupSelection === 'groupByRepos'")
               .summary-charts__title--index {{ i+1 }}
               .summary-charts__title--repo {{ repo[0].repoName }}
-              .summary-charts__title--contribution.mini-font(title="Total contribution of group") [{{ getGroupTotalContribution(repo) }} lines]
+              .summary-charts__title--contribution.mini-font(title="Total contribution of\
+               group") [{{ getGroupTotalContribution(repo) }} lines]
```

Comment:
hmm, it doesn't seems to be enforcing the indentation of line wrap. Is there a rule for this?
line wraps are supposed to be 4 spaces

Reply:
Nope. Can't seem to find a rule for line wraps.

Reply:
This line can be broken in a more appropriate place.

Reply:
Break before `[{{`